karma.conf.js

```diff
@@ -28,7 +30,15 @@ module.exports = function(config) {
     webpack: {
       //mode: 'production',
       mode: 'development',
-      devtool: 'inline-source-map'
+      devtool: 'inline-source-map',
+      node: {
+        crypto: false
+      },
+      resolve: {
+        alias: {
+          crypto$: path.resolve(__dirname, 'no-crypto.js')
+        }
```

Comment:
This isn't a thing anymore, right?

Reply:
I suspect the crypto stuff could be removed entirely, but here's what happens in another module:

https://github.com/digitalbazaar/x25519-key-agreement-key-2020/blob/main/karma.conf.js#L28-L39

Reply:
I have no idea... @davidlehn @dmitrizagidulin ?

Reply:
It is still confusing what to do with all this!  Probably should align things and bump webpack here to 5.x, make adjustments as needed, and change to what that other package is doing.